src/sensor_fusion.c

```diff
@@ -0,0 +1,346 @@
+/**
+ * \brief Function Definition Library to implement Sensor Fusion Algorithm
+ * \author jaspalsingh@cmail.carleton.ca
+ * \version 1.0
+ * \date 2019-12-01
+ */
```

Comment:
You should include a detailed description here about the file.

Reply:
Fixed in recent commits to dev branch